### PR TITLE
i18n(es): Translated `space.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `space.mdx` to spanish 🚀

### CODE REVIEW

File: src/content/docs/es/guides/deploy/space.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+[Deta Space](https://deta.space/developers) es una computadora personal que vive en la nube — una "nube personal". Puedes construir y ejecutar aplicaciones para ti mismo en tu propio "Espacio". Puedes publicar las aplicaciones que has construido y se ejecutarán para personas en todo el mundo.
+
+Esta guía incluye instrucciones paso a paso para construir sitios en Space. Ambos estáticos y rendereados del lado del servidor (con el [adaptador `@astrojs/node`](/es/guides/integrations-guide/node/)) son compatibles con Astro.
+
+## Prerrequisitos
+
+Para enviar un sitio Astro a Space, asegúrate de que primero:
+
+- Crea una [cuenta Space](https://deta.space).
+- Instala el [CLI de Space](https://deta.space/docs/en/basics/cli) e inicia sesión.
```
Comment (raul-repos) [inline src/content/docs/es/guides/deploy/space.mdx:19]:
```suggestion
Para enviar un sitio Astro a Space, asegúrate de que primero:

- Tienes una [cuenta Space](https://deta.space).
- Instalas el [CLI de Space](https://deta.space/docs/en/basics/cli) e inicias sesión.
```

File: src/content/docs/es/guides/deploy/space.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+:::note
+El CLI de Space intentará detectar automáticamente la configuración de tu aplicación. Acepta la configuración sugerida y sigue las instrucciones a continuación, dependiendo del tipo de aplicación Astro que quieras desplegar.
+:::
+
+
+## Configuración del proyecto
+
+### Sitio estático
+
+Realiza los siguientes cambios en el archivo `Spacefile` en la raíz de tu proyecto generado por el CLI de Space.
```
Comment (raul-repos) [inline src/content/docs/es/guides/deploy/space.mdx:36]:
```suggestion
Realiza los siguientes cambios en el archivo `Spacefile` en la raíz de tu proyecto generado por el CLI de Space:
```

File: src/content/docs/es/guides/deploy/space.mdx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    engine: static
+    commands:
+      - npm run build
+    serve: dist
+```
+
+### Sitio renderizado del lado del servidor
+
+
+Make the following changes to the `Spacefile` file at the root of your project generated by the Space CLI:
```
Comment (raul-repos) [inline src/content/docs/es/guides/deploy/space.mdx:57]:
```suggestion
Realiza los siguientes cambios en el archivo `Spacefile` en la raíz de tu proyecto generado por el CLI de Space:
```

File: src/content/docs/es/guides/deploy/space.mdx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+```bash
+space push
+```
+
+Esto ejecutará el proceso de construcción y creará una nueva [instancia de la aplicación Space](https://deta.space/docs/en/basics/revisions#testing-changes) donde puedes acceder a tu aplicación Astro.
+
+Por defecto las aplicaciones Space son privadas y solo son accesibles para ti.
+
+Si quieres que tu aplicación esté disponible para otros, puedes usar [Rutas públicas](https://deta.space/docs/en/reference/spacefile#public_routes) para hacer partes de tu aplicación pública. O puedes [crear una versión](https://deta.space/docs/en/basics/releases) para que otros instalen tu aplicación en su propia nube personal.
```
Comment (raul-repos) [inline src/content/docs/es/guides/deploy/space.mdx:90]:
```suggestion
Si quieres que tu aplicación esté disponible para otros, puedes usar [Rutas públicas](https://deta.space/docs/en/reference/spacefile#public_routes) para hacer partes de tu aplicación públicas. También puedes [crear una versión](https://deta.space/docs/en/basics/releases) para que otros instalen tu aplicación en su propia nube personal.
```

**Review by raul-repos — CHANGES_REQUESTED** [review]

**Comment by dreyfus92** [comment]
Muchas gracias por el review @raul-repos 🚀

**Review by Yan-Thomas — APPROVED** [review]
LGTM, one more getting merged today, yay! Thanks @dreyfus92 & @raul-repos, I really appreciate the continued effort here 🙌

**Pull request merged**